index.html

```diff
@@ -0,0 +1,252 @@
+         <section class="about">
+          <div class="about-content">
+          <h1> About me</h1>
+          <p>Hello I’m a software developer!
+               I can help you build a product , feature
+                or website Look through some of my work 
+               and experience! If you like what you
+               see and have a project you need coded, 
+                don’t hestiate to  contact me.</p> 
+          
+                <button type="button" class="get">Get My Resume</button>
+
+              </div>
+
+                <img class="about-img" src="images/about-desktop.png" alt="">
+                <span class="img-about"></span>
+                <hr class="about-hr">
+               <div class="about-list" > 
+                <div class="list-content">
+                <img class="list-icon"   src="images/Rectangle.png" alt="">
+                <span class="lang">Language</span>
+                <ul class="container">
+                  <li>Javascript</li>
+                  <li>Ruby</li>
+                  <li>HTML</li>
+                  <li>Css</li>
+                </ul>
+              </div>
+            <div class="list-content">
+                 <img  class="list-icon" src="images/Square.png" alt="">
+                <span class="frame">Frameworks</span>
+                <ul class="container">
+                  <li>Bootstrap</li>
+                  <li>Ruby on Rails</li>
+                  <li>RSPec</li>
+                  <li>CapyBare</li>
+                  <li>Selenium</li>
+                </ul>
+              </div>
+            <div class="list-content">
+                <img class="list-icon1" src="images/Circle.png" alt="">
+                <span class="skill">Skills</span>
+                <ul class="container">
+                  <li>Codekit</li>
+                  <li>Github</li>
+                  <li>Codepen</li>
+                  <li>Gitlab</li>
+                  <li>Terminal</li>
+                </ul>
+              </div>
+              </div>
+      </section> 
+      <section class="contact container">
+        <p class="title"> I'm always <br> 
+            interested in hearing about new projects,so if you'd
+             like to chat please get in touch.</p>
+         <form action="https://formspree.io/f/mdojzzgj" method="post">
+             
+          <input type="text" class="item item1" name="firstName"  placeholder="First Name" required   maxlength="30">
+          <input type="text" class="item item2" name="lastName" placeholder="Last Name" required   maxlength="30">
+            <input type="email" class="item" placeholder="Email address" name="email" required >
+           
+            <textarea  name="message" id="tetarea" class="item textarea" cols="20" rows="7" placeholder="Hello Mariane. Glad to see you |"   maxlength="500" required></textarea>
+           
+           
+          <button type="submit" class="touch">Get in touch</button>
+          
+         </form>
```

Comment:
- The contact form in your mobile version doesn't exactly match what is in the figma template. Please look into fixing this issue 🙏

Your project( Play pay attention to the text and the alignment of the submit button):
<img width="152" alt="contact student" src="https://user-images.githubusercontent.com/73607512/194556305-2e505e6e-4949-40a5-93ce-022746c5abdb.PNG">

Figma template:
<img width="138" alt="contact figma" src="https://user-images.githubusercontent.com/73607512/194556348-8f99f3f7-c889-41a6-8b98-a4fa39edcd96.PNG">

Reply:
Fixed.